src/OpenLoco/Title.cpp

```diff
@@ -1,4 +1,4 @@
-#include "Title.h"
+﻿#include "Title.h"
```

Comment:
Byte-order mark insertion?

Reply:
Yeah sorry about that, #777 will prevent this in the future.